slrealizer/realize_sl.py

```diff
@@ -0,0 +1,313 @@
+#from __future__ import absolute_import
+#from __future__ import division
+#from __future__ import print_function
+
+import numpy as np
+from utils.utils import *
+from utils.constants import *
+import pandas as pd
+import matplotlib.pyplot as plt
+import pylab
+import matplotlib
+#import skimage
+import random
+#import corner
+#import dropbox
+#import extract_corner
+import om10
+import galsim
+from astropy.utils.console import ProgressBar
+#from corner import corner
+
+class SLRealizer(object):
+
+    '''
+    Generates the toy catalog, plots the lensed system, and deblends sources
```

Comment:
With the new and improved inheritance scheme, does this docstring need updating?

Reply:
Updated.